pages/27.Marketplace/default.en.md

```diff
@@ -129,24 +142,28 @@ There are options that will allow you to filter or go to next pages. The full li
   -h, --help             Display this help message
 ```
 
-Example usage how to search for Captcha Plugin: `bin/console mautic:marketplace:list --filter=captcha`
+Example usage how to search for a Captcha Plugin: `bin/console mautic:marketplace:list --filter=captcha`
 
 ### Planned features
 
-The Marketplace is lacking some important features. In the text above there are mentions of several planned features. Most of the features are waiting for the Mautic 4 release and its main features (Composer v2 support, new directory structure) and therefore can be developed when all these features come together.
+The Marketplace is lacking some important features. In the preceding text there are mentions of several planned features. Most of the features are waiting for the Mautic 4 release and its main features (Composer v2 support, new directory structure).
```

Comment:
Small nit: Composer v2 support has been in Mautic since 4.0, so I think we can get rid of the "lacking some important features" 😊

Reply:
I've tweaked the text, see what you think!

> ### Planned features
> 
> Watch out for more features in future releases including:
>